docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -19,6 +19,10 @@ The main features of the Media Controller API include:
 
   You can [send a request](#managing-requests) from the client to the server to modify various playback attributes. In the server, you can set up a listener to react to the client request and perform the requested task.
 
+- Set server icon URI
```

Comment:
To maintain consistency, please rephrase as:
Setting the server icon URI

Reply:
done